modules/meilisearch/meilisearch_test.go

```diff
@@ -0,0 +1,86 @@
+package meilisearch_test
+
+import (
+	"context"
+	"io"
+	"net/http"
+	"net/url"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/modules/meilisearch"
+)
+
+func TestMeilisearch(t *testing.T) {
+	ctx := context.Background()
+
+	ctr, err := meilisearch.Run(ctx, "getmeili/meilisearch:v1.10.3")
+	testcontainers.CleanupContainer(t, ctr)
+	require.NoError(t, err)
+
+	address, err := ctr.Address(ctx)
+	require.NoError(t, err)
+
+	client := http.DefaultClient
+
+	req, err := http.NewRequest(http.MethodGet, address, nil)
+	require.NoError(t, err)
+
+	resp, err := client.Do(req)
+	require.NoError(t, err)
+	defer resp.Body.Close()
+}
+
+func TestMeilisearch_WithDataDump(t *testing.T) {
+	ctx := context.Background()
+
+	ctr, err := meilisearch.Run(ctx, "getmeili/meilisearch:v1.10.3",
+		meilisearch.WithDumpImport("testdata/movies.dump"),
+		meilisearch.WithMasterKey("my-master-key"))
```

Comment:
nit: I find closing the brace on a new line when using multi line syntax like helps be understand the block better.
```suggestion
	ctr, err := meilisearch.Run(ctx, "getmeili/meilisearch:v1.10.3",
		meilisearch.WithDumpImport("testdata/movies.dump"),
		meilisearch.WithMasterKey("my-master-key"),
	)
```